src/app/_graphql/podcastepisodes.ts

```diff
@@ -0,0 +1,55 @@
+// TODO: Figure out refering to contributors and media files.
+
```

Comment:
Rename to `podcastEpisodes.ts`, to keep consistency ;)

Reply:
done ✅